.github/autolabeler.yml

```diff
@@ -0,0 +1,83 @@
+
```

Comment:
Do I need to add the ASF iceberg license at the top of the file manually or will that happen on merge or in some other way?

Reply:
Yes,  I think we should attach the license header in this file manually.

Reply:
Yes, we need the ASF header.

Reply:
Header has been added.